.github/workflows/ci.yml

```diff
@@ -2,34 +2,37 @@ name: CI
 
 on:
   push:
-  pull_request_target:
+  pull_request:
   schedule:
     - cron: "6 20 * * 6"
 
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: ${{matrix.image}}
     strategy:
       fail-fast: false
       matrix:
-        image:
-          - ruby:2.5.0
-          - ruby:2.5
-          - ruby:2.6.0
-          - ruby:2.6
-          - ruby:2.7.0
-          - ruby:2.7
-          - jruby:9.2
-        include:
-          - image: ruby:rc
-            continue-on-error: true
+        ruby:
+          - 2.5
+          - 2.6
+          - 2.7
+          - ruby-head
```

Comment:
We don't want to break the build on unstable rubies failures. These are only so that we knew we have some work before we can support the next release. There's also a chance that unstable rubies might have bugs.

Reply:
Unfortunately that feature doesn't really exist on GitHub Actions.

I suppose I could move the RCs to another pipeline, and have a || true, but then it will show as green on GitHub so the value is limited.

Reply:
It does exist. It's not obvious but here's how it's done. You include a duplicate entry in the matrix and add `continue-on-error: true` to it. See Line 24 in the current config.